core/src/mindustry/editor/TileCopy.java

```diff
@@ -0,0 +1,31 @@
+package mindustry.editor;
+
+import mindustry.world.Tile;
+
+import java.util.ArrayList;
+
+public class TileCopy{
+    public ArrayList<ArrayList<Tile>> data=new ArrayList<>();
```

Comment:
Why is this a 2D arraylist when it could be a 2D array?

Reply:
Hh sorry i absolutly forgot about that.Now am changing it and adding more new features so you can deny this pull request ill do another one later. Is it ok if i also make shortcuts for editor like selecting pencil by clicing P and so on?